eng/pipelines/mono/templates/xplat-job.yml

```diff
@@ -58,6 +58,8 @@ jobs:
       agentOs: FreeBSD
     ${{ if eq(parameters.osGroup, 'OSX') }}:
       agentOs: MacOS
+    ${{ if eq(parameters.osGroup, 'iOS') }}:
+      agentOs: MacOS
```

Comment:
Could we just merge the two in one condition? OSX and iOS using `in` conditional?